src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -69,6 +86,28 @@ export default class ReplyField extends React.Component<Props, State> {
         this.saveCursorPosition();
     }
 
+    getItemList = (editorState: EditorState): Collaborator[] => {
```

Comment:
These components seem to be doing a lot. Can we break things up to have more specific responsibilities?

Reply:
> These components seem to be doing a lot. Can we break things up to have more specific responsibilities?

If we want `Editor` and `PopupList` to be sibling, the `handleKeyDown` and `getItemList` have to live in their common parent, which is this file.